Temporary_PHPCompatibility_Test.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Should tell me return void is not available in PHP 5.6
+class Foo {
+	public function do_nothing(): void {
+
+	}
+}
+
+// Should tell me anonymous class are not available in PHP 5.6
+$a = new class {}
+
+// Should tell me "create_function()" has been deprecated on PHP 7.2
+create_function('', '');
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 1b23e12727e46bff82706f1484d9b863</sub>